plugins/keyboard-navigation/src/gesture_monkey_patch.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Overrides methods on Blockly.Gesture in order to allow users
+ * to move the cursor to blocks or the workspace using shift click.
+ * TODO(google/blockly#4584): We do not have a way to do this currently without
+ * monkey patching Blockly.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+import * as Blockly from 'blockly/core';
+
+
+const oldDoWorkspaceClick = Blockly.Gesture.prototype.doWorkspaceClick_;
+
+/**
+ * Execute a workspace click. When in accessibility mode shift clicking will
+ * move the cursor.
+ * @param {!Event} e A mouse up or touch end event.
+ * @this {Blockly.Gesture}
+ * @override
+ */
+Blockly.Gesture.prototype.doWorkspaceClick_ = function(e) {
+  const ws = this.creatorWorkspace_;
```

Comment:
My preference is that this set is after `oldDoWorkspaceClick` call.

Reply:
done